app/models/place.rb

```diff
@@ -29,4 +33,52 @@ def self.prefectures_with_key
       }
     end
   end
+
+  def self.create_index!(options={})
+    client = __elasticsearch__.client
+    settings index: {
+               analysis: {
+                 tokenizer: {
+                   ngram_tokenizer: {
+                     type: 'nGram',
+                     min_gram: '2',
+                     max_gram: '3',
+                     token_chars: ['letter', 'digit']
+                   }
+                 },                 
+                 analyzer: {
+                   ngram_analyzer: {
+                     tokenizer: "ngram_tokenizer"
+                   },
+                   kuromoji_analyzer: {
+                     tokenizer: 'kuromoji_tokenizer',
+                     filter:    ['kuromoji_baseform'],
```

Comment:
__[rubocop]__
* Avoid comma after the last item of a hash. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas)